data/lit0001/pw0001/__cts__.xml

```diff
@@ -1,14 +1,14 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:mayaLit:lit0001" urn="urn:cts:mayaLit:lit0001.pw0001" xml:lang="quc">
   <ti:title xml:lang="quc">Popol Wuj</ti:title>
   
-  <ti:edition workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-quc:">        
+  <ti:edition workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-quc">        
     <ti:label xml:lang="quc">Popol Vuh</ti:label>
     <ti:description xml:lang="eng">Ximenez copy, K'iche' column</ti:description>
    </ti:edition>         
   
-    <ti:edition workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-spa:">        
-      <ti:label xml:lang="quc">Popol Vuh</ti:label>
+    <ti:translation workUrn="urn:cts:mayaLit:lit0001.pw0001" urn="urn:cts:mayaLit:lit0001.pw0001.popolwuj-spa" xml:lang="spa">        
+      <ti:label xml:lang="spa">Popol Vuh</ti:label>
```

Comment:
This should be the language of the label itself so `quc` is correct, I think

Reply:
@jtauber good catch; was left over from my attempts to resolve metadata issues.  Have reverted via 4030c79.